packages/angular/cli/src/commands/analytics/info/cli.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { tags } from '@angular-devkit/core';
+import { Argv } from 'yargs';
+import { analyticsConfigValueToHumanFormat } from '../../../analytics/analytics';
+import {
+  CommandModule,
+  CommandModuleImplementation,
+  Options,
+} from '../../../command-builder/command-module';
+import { getWorkspaceRaw } from '../../../utilities/config';
+
+export class AnalyticsInfoCommandModule
+  extends CommandModule
+  implements CommandModuleImplementation
+{
+  command = 'info';
```

Comment:
I am not uncertain about the command to output the info `$ ng analytics info` vs `$ ng analytics`. I used the former as I found it to be more explicate.

Reply:
I find `enabled` to be rather similar to `enable` which might be confusing. Actually, I'd prefer if we the turn off/on the commands where `ng analytics enable` / `ng analytics disable`.


My reasoning to use `info` was that I future we might want to add additional details such as the uuid etc... and eventually we can use the same command syntax for `ng cache`. `ng cache info`,  `ng cache on` etc...

Reply:
I'm fine with `info`. Agreed that it would be great to be consistent across subcommands if possible.